fl4health/utils/partitioners.py

```diff
@@ -99,9 +126,9 @@ def partition_label_indices(self, label: T, label_indices: torch.Tensor) -> Tupl
         partitioned_indices = list(torch.split(shuffled_indices, num_samples_per_partition))
 
         # Dropping the last partition as they are "excess" indices
-        return partitioned_indices[:-1], min_samples
+        return partitioned_indices[:-1], min_samples, partition_allocations
 
-    def partition_dataset(self, original_dataset: D, max_retries: int = 5) -> List[D]:
+    def partition_dataset(self, original_dataset: D, max_retries: int = 5) -> Tuple[List[D], Dict[T, np.ndarray]]:
```

Comment:
The documentation on the return here still needs to be updated to reflect the `Dict[T, np.ndarray]` that has been added I think.

Reply:
I have added a comment as
` np.ndarray: The Dirichlet distribution used to partition the data points.`
to reflect the changes. Do I need to add anything extra?

Reply:
I think that's in a different functions documentation (if I'm reading everything correctly). Right now, the documentation for this function has the following:
```
        Returns:
            List[D]: The partitioned datasets, length should correspond to self.number_of_partitions

        """
```
which doesn't reflect the return type that we're using for annotation above.